embedding-sdk/unauthenticated/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Charts Embedded SDK Demo</title>
+    <meta charset="UTF-8" />
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Heebo&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" type="text/css" href="assets/styles.css" />
+    <link rel="stylesheet" type="text/css" href="assets/toggle.css" />
+  </head>
+
+  <body>
+    <main>
+      <div id="nav">
+        <label
+          >REFRESH CHART&nbsp&nbsp
+          <button id="refresh">⟳</button>
+        </label>
+        <label
+          >REFRESH INTERVAL&nbsp&nbsp
+          <select id="refresh-interval">
+            <option value="0">None</option>
+            <option value="10">10 Seconds</option>
+            <option value="10">1 &nbsp&nbspMinute</option>
+            <option value="100">10 Minutes</option>
+            <option value="600">1 &nbsp&nbspHour</option>
+          </select>
+        </label>
+        <label
+          >FILTER BY COUNTRY&nbsp&nbsp
+          <select id="country-filter">
+            <option value="">--All--</option>
+            <option value="Australia">Australia</option>
+            <option value="Brazil">Brazil</option>
+            <option value="Canada">Canada</option>
+            <option value="China">China</option>
+            <option value="Hong Kong">Hong Kong</option>
+            <option value="Portugal">Portugal</option>
+            <option value="United States">United States</option>
+          </select>
+        </label>
+        <label class="theme-label"
+          >THEME&nbsp&nbsp
+          <label class="switch">
+            <input id="themeSwitch" type="checkbox" />
+            <span class="slider round"></span>
+          </label>
+          <span id="themeEmoji" class="theme-label"> ☀️ </span>
+        </label>
+      </div>
+      <div style="display:flex">
+        <div id="chart"></div>
+        <div class="chartInfo">
+          <h3 style="text-align:center;color:#89989b;">CHART DATA</h3>
+          <table class="chartInfoTable">
+            <tr>
+              <td class="tableHeader">Current Refresh Interval</td>
+              <td id="currentRefreshInterval">None</td>
+            </tr>
+            <tr>
+              <td class="tableHeader">Current Theme</td>
+              <td id="currentTheme">light</td>
+            </tr>
+            <tr>
+              <td class="tableHeader">Current Filter</td>
+              <td id="currentFilter">None</td>
+            </tr>
+          </table>
```

Comment:
```suggestion
          <table>
            <tr>
              <th>Current Refresh Interval</th>
              <td id="currentRefreshInterval">None</td>
            </tr>
            <tr>
              <th>Current Theme</th>
              <td id="currentTheme">light</td>
            </tr>
            <tr>
              <th>Current Filter</th>
              <td id="currentFilter">None</td>
            </tr>
          </table>
```
you can simplify the styles here by using the semantically correct `th` tag instead